### DISCUSSION TITLE
It doesn’t work for me I am M1 chip [Oct 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4216] [Discord Threads]
> 🙋‍♂️ **tosmatt**

**Comment by o1lab** [comment]
> 🙋‍♂️ **mart**  
#### docker buildx create --name m1   
#### docker buildx use m1   
####    
#### docker buildx build . --tag $tag -o type=image --platform=linux/arm64,linux/amd64  
#  
> 🙋‍♂️ **mart**  
#### ?  
#  
> 🙋‍♂️ **tosmatt**  
#### Let me try  
#  
> 🙋‍♂️ **tosmatt**  
####   
![image0](https://cdn.discordapp.com/attachments/1026803005053480970/1026805294543683594/unknown.png)  
#  
> 🙋‍♂️ **tosmatt**  
####   
![image0](https://cdn.discordapp.com/attachments/1026803005053480970/1026805419039006750/unknown.png)  
#  
> 🙋‍♂️ **tosmatt**  
#### It still doesn't work. It's the same error  
#  
> 🙋‍♂️ **Alsu**  
#### I did : npm install --save-dev @ava/typescript@2 and npm install --save-dev @ava/typescript, it solved it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1026803005053480970)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
